content/applications/marketing/email_marketing/lost_leads_email.rst

```diff
@@ -0,0 +1,337 @@
+   records for the email campaign, and will not include a complete or accurate lost leads audience.
+
+Click the :guilabel:`(Add Branch)` icon to the right of the latest rule's row to add a pair of
+:guilabel:`any of` rules. In the top rule's first field, select the :guilabel:`Active` parameter
+from the drop-down menu. The rule will then automatically fill out to read: :guilabel:`Active` *is*
+`set`.
+
+For the first field of the bottom rule, once more, select :guilabel:`Active` from the drop-down
+menu and, this time, select :guilabel:`is not` from the operator drop-down menu, so the rule reads:
+:guilabel:`Active` *is not* `set`.
+
+.. image:: lost_leads_email/active.png
+   :align: center
+   :alt: A pair of Match Any Of filter rules that include both active and inactive leads.
+
+Add body content
+================
+
+Now, with the domain section of the email campaign complete, create the body content of the email
+using any of the premade stylized templates, or choose between the :guilabel:`Plain Text` or
+:guilabel:`Start From Scratch` options for more granular control. For more information, refer to the
+*Email Marketing* :ref:`documentation on how to create an email <email_marketing/create_email>`.
+
+.. tip::
+   To save the set of filters for later use, click :guilabel:`Save as Favorite Filter 💾`, enter a
+   name (such as `Lost Leads`), and click :guilabel:`Add`.
+
+   .. image:: lost_leads_email/favorite-filter.png
+      :align: center
+      :alt: The Save as Favorite Filter pop-up can save the lost leads criteria for later.
+
+Send or schedule
+================
+
+Once all the components of the email campaign are complete:
+
+- Click the purple :guilabel:`Send` button at the top-left of the form to immediately send the
+  email;
+
+  OR
+- Click gray :guilabel:`Schedule` button, located to the right of the :guilabel:`Send` button, to
+  send the email at a future date and time.
+
+.. tip::
+   Consider using A/B Testing to send an alternate version of the email to a percentage of the
+   target leads. This can help determine what subject lines and body content produce the best
+   click-through rates before sending a final version to the remaining leads.
+
+   To do so, open the :guilabel:`A/B Tests` tab and check the box next to :guilabel:`Allow A/B
+   Testing`. Adjust the parameters as needed, and click :guilabel:`Create an Alternative Version`.
+
+   .. image:: lost_leads_email/ab-testing.png
+      :align: center
+      :alt: The A/B Tests tab with the Allow A/B Testing box checked to create an alternate version.
+
+.. _email_marketing/select_lost_reasons:
+
+Select appropriate lost reasons
+===============================
+
+When a lead is marked as lost, Odoo recommends selecting a *Lost Reason* to indicate why the
+opportunity did not result in a sale. Doing so keeps the pipeline organized and reporting data
+accurate, and also generates potential to follow up with the lead in the future.
```

Comment:
```suggestion
accurate. This tactic also generates the potential to follow-up with that lead in the future.
```

Reply:
I think the solution here actually is to remove a word, so I did this instead:

```suggestion
accurate, and generates potential to follow up with the lead in the future.`
```